BCH-Wunder-Project-team-4/Wunder.io
# Job listing content type

## Link to ticket:

[Trello ticket](https://trello.com/c/OYSa2INI/71-content-type-job-listing)

## Changes proposed in this PR:

Adding a new content type 'job listing' based on the 'page' content type. 

## How to test:

1. Do 'lando drush cim' in the terminal to apply the Drupal configuration.
2. Check the Drupal admin panel, it should have the new content type working.
